### PR TITLE
Regenerate code for release

### PR DESCRIPTION
**Pull request by OGKevin**
[//]: # (Thanks for opening this pull request! Before you proceed please make sure that you have an issue that explains what this pull request will do.
         Make sure that all your commits link to this issue e.g. "My commit. \(bunq/sdk_java#<issue nr>\)".
         If this pull request is changing files that are located in "com.bunq.sdk.model.generated" then this pull request will be closed as these files must/can only be changed on bunq's side.)
         
## This PR closes/fixes the following issues:
[//]: # (If for some reason your pull request does not require a test case you can just mark this box as checked and explain why it does not require a test case.)
 - Closes bunq/sdk_java#65
    - [x] Tested

### CODE REVIEW

**Comment by OGKevin** [comment]
@patrickdw1991 please 👀

File: src/main/java/com/bunq/sdk/examples/CardDebitExample.java [hunk 1]
```diff
@@ -20,19 +22,27 @@
   private static final String EMAIL_YOUR_COMPANY = "at@at.at"; // Put your user email here
   private static final String POINTER_NAME_TEST = "test pointer";
   private static final int USER_ITEM_ID = 0; // Put your user ID here
+  private static final int MONETARY_ACCOUNT_ID = 0; // Put your monetary account ID here
   private static final int NUM_BITS_20_DECIMAL_DIGITS = 66;
   private static final int RADIX_DIGITS_AND_LATIN_LETTERS = 36;
+  private static final String CARD_PIN_ASSIGNMENT_TYPE_PRIMARY = "PRIMARY";
```
Comment (patrickdw1991) [inline src/main/java/com/bunq/sdk/examples/CardDebitExample.java]: Is there a reason why this is not an enum? Or is that impossible to generate?

Comment (OGKevin) [reply src/main/java/com/bunq/sdk/examples/CardDebitExample.java]: Generating ENUMs is also on the list. But its kind of impossible however. Will PM you the details.

**Comment by patrickdw1991** [comment]
@OGKevin All good :)

**Comment by OGKevin** [comment]
@andrederoos please have a look

**Pull request merged**